File: Beecrowd/Beginner./1046.cpp
#include <iostream>
using namespace std;

int main() {
    int start, end, duration;

    cin >> start >> end;

    if (start < end) {
        duration = end - start;
    } else {
        duration = 24 - start + end;
    }

    cout << "O JOGO DUROU " << duration << " HORA(S)" << endl;

    return 0;
}

File: Beecrowd/1. Beginner/1046/1046.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);

        int start, end, duration;

        start = input.nextInt();
        end = input.nextInt();

        if(start < end) {
            duration = end - start;
        } else {
            duration = 24 - start + end;
        }

        System.out.println("O JOGO DUROU " + duration + " HORA(S)");

        input.close();
    }
}